paths/electron/starting-with-electron/02-electron-boilerplate.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Create your app
+permalink: /electron/create-an-app/create-your-app/
+next-page: /electron/create-an-app/push-your-project-to-github/
+facilitator: false
+sidebar:
+  nav: "create-an-app-in-electron"
+main-content: |
+
+  To begin, we will set up a base electron app using a boilerplate called [electron-forge](https://electronforge.io/).
```

Comment:
@malept Okay. Do you have another recommendation? Would 'npm package' be better?